forest/cli/src/cli/send_cmd.rs

```diff
@@ -129,6 +129,7 @@ pub struct SendCommand {
     from: Option<Address>,
     target_address: Address,
     /// token amount in attoFIL
+    // BUG?(aatifsyed): sure the amount is in FIL??
```

Comment:
`FILAmount` is just a wrapper around `TokenAmount` that supports human-readable formats. We recently stopped using `fvm_shared::TokenAmount` and are now using `forest_shim::econ::TokenAmount`. This means we can eliminate `FILAmount` and directly define `from_str()` for `TokenAmount`.

Reply:
Yes, so I think this is a doc bug:
`token amount in attoFIL` should be `token amount in FIL`